hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1969,6 +1971,18 @@ public String getCanonicalServiceName() {
     return (dtService != null) ? dtService.toString() : null;
   }
 
+  @Override
+  public byte[] postRPCReq(byte[] payload, boolean emptyResp) {
+    if (emptyResp) {
+      return null;
+    }
+    if (payload == null || payload.length == 0) {
+      return RandomUtils.nextBytes(respBytesSize);
+    }
+    return Arrays.copyOf(payload, payload.length);
```

Comment:
I am puzzled here. Shouldn't the RpcClient send a message to OM?

Reply:
Make sense. Checking!